packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleVideoViewer/SingleVideoViewerContainer.js

```diff
@@ -153,20 +199,55 @@ class SingleVideoViewerContainer extends React.Component {
     //   return null
     // }
 
+    const transformLayer = React.createRef()
+    const svg = this.videoViewer.current
+    const transform = ``
+    const getScreenCTM = () => transformLayer.current.getScreenCTM()
+
+    const enableDrawing =
+      loadingState === asyncStates.success && enableInteractionLayer
+
     return (
-      <div>
-        <div>
+      <>
+        <ScreenContainer>
           <SingleVideoViewer
             playerRef={this.handlePlayerRef}
-            url={vid}
+            url={vid.src}
             isPlaying={isPlaying}
             playbackRate={playbackRate}
             onProgress={this.handleVideoProgress}
             onDuration={this.handleVideoDuration}
             onEnded={this.handleVideoEnded}
           />
-          {/* Drawing layer here */}
-        </div>
+
+          {/* Drawing Layer */}
+          <DrawingLayer>
+            <Box animation='fadeIn' overflow='hidden'>
+              <SVGContext.Provider value={{ svg, getScreenCTM }}>
+                <svg
+                  ref={this.videoViewer}
+                  focusable
+                  onKeyDown={onKeyDown}
+                  tabIndex={0}
+                  viewBox={`0 0 ${naturalWidth} ${naturalHeight}`}
+                >
+                  {/* {title?.id && title?.text && (
+                  <title id={title.id}>{title.text}</title>
+                )} */}
+                  <g ref={transformLayer} transform={transform}>
+                    {enableInteractionLayer && (
+                      <InteractionLayer
+                        scale={1}
```

Comment:
I think you can set the scale here to something like `clientWidth / naturalWidth`, which will scale the drawn marks down to size. At the moment, they're drawn with oversized lines and points.